README.md

```diff
@@ -1,78 +1,280 @@
+[{path_name}/matrixone/test/cases/benchmark/tpch/01_DDL/01_create_table.sql] COST : 0.226s, TOTAL :11, SUCCESS :11, FAILED :0, IGNORED :0, ABNORAML :0, SUCCESS RATE : 100%
+[{path_name}/matrixone/test/cases/benchmark/tpch/02_LOAD/02_insert_customer.sql] COST : 0.357s, TOTAL :16, SUCCESS :16, FAILED :0, IGNORED :0, ABNORAML :0, SUCCESS RATE : 100%
 ```
 
-If you'd like to adjust the test range, you can just change the `path` parameter of `run.yml`. 
+|Report Keywords|Description|
+|---|---|
+|TOTAL|the total number of executed test cases (SQL)|
+|SUCCESS|The total number of successfully executed test cases(SQL)|
+|FAILED|the total number of failed executed test case(SQL)|
+|IGNORED| the total number of ignored executed test cases (SQL), especially with the `--bvt:issue` tag test cases (SQL)|
+|ABNORAML|the total number of abnormal executed test cases (SQL), such as the execution of MatrixOne can't determine the actual result is a system exception or *.result* file parsing error, etc|
+|SUCCESS RATE|success rate: SUCCESS/(TOTAL - IGNORED)|
+
+* An example of *error.txt* file looks like this:
 
-And you can also specify some parameters when executing the command `run.sh`,parameters are as followings:
 ```
--p  set the path of test cases needed to be executed by mo-tester,default value is configured by the `path` in `run.yaml`
--m  set the method that mo-tester will run with,default value is configured by the `method` in `run.yaml`
--t  set the type of the format that mo-tester execute the sqlcommand in,default value is configured by the `type` in `run.yaml`
--r  set The success rate that test cases should reach,default value is configured by the `rate` in `run.yaml`
--i  set the including list, and only script files in the path which name contain one of the list will be excuted,if more than one,seperated by `,`,if not specified,refers to all cases included
--e  set the excluding list, and script files in the path which name contain one of the list will not be excuted,if more than one,seperated by `,`,if not specified,refers to none of cases excluded
--g  means SQL commands which is marked with [bvt:issue] flag will not be executed,this flag starts with [-- @bvt:issue#{issueNO.}],and ends with [-- @bvt:issue],eg:
-    -- @bvt:issue#3236
-    select date_add("1997-12-31 23:59:59",INTERVAL "-10000:1" HOUR_MINUTE);
-    select date_add("1997-12-31 23:59:59",INTERVAL "-100 1" YEAR_MONTH);
-    -- @bvt:issue
-    Those two sql commands are associated with the issue#3236,and they will not been executed in bvt test,until the flag is removed when the issue#3236 is fixed.
-    
--n  means the meta data of the resultset will be ignored when comparing the result
-Examples:
-bash run.sh -p case -m run -t script -r 100 -i select,subquery -e substring -g
+[ERROR]
+[SCRIPT   FILE]: cases/transaction/atomicity.sql
+[ROW    NUMBER]: 14
+[SQL STATEMENT]: select * from test_11 ;
+[EXPECT RESULT]:
+c	d
+1	1
+2 2
+[ACTUAL RESULT]:
+c	d
+1	1
 ```
```

Comment:
add explain the meaning of expect result and actual result，maybe a little better